src/lib/components/form/BuildForm.svelte

```diff
@@ -158,25 +183,33 @@
   async function onsubmit(event: SubmitEvent): Promise<void> {
     event.preventDefault();
 
-    errorMessage = "";
+    issues = [];
     saving = true;
 
-    // Temporary fake load times
-    await new Promise((res) => setTimeout(res, 500));
-
     try {
-      const response = await api(path, {}, null, {
+      const response = (await api(path, {}, null, {
         method,
         body: JSON.stringify(build),
-      });
+      })) as { newBuild: StadiumBuild };
 
       if (!response) throw new Error("Something went wrong while saving");
 
-      // TODO: Redirect to created build
+      const { newBuild } = response;
+
+      await goto(`/build/${newBuild.id}-${toSlug(newBuild.buildTitle)}`);
```

Comment:
There is a routes util now
```suggestion
      await goto(buildPath(build));
```
imported from `$lib/utills/routes`